apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/Logging/GoogleLoggerTest.cs

```diff
@@ -359,6 +359,26 @@ public void Log_Labels()
             mockConsumer.VerifyAll();
         }
 
+        [Fact]
```

Comment:
This test might not be needed since is better if we don't silently remove labels.